lib/driver.js

```diff
@@ -652,10 +652,8 @@ class XCUITestDriver extends BaseDriver {
       this.logEvent('customCertInstalled');
     }
 
-    if (this.opts.app) {
-      await this.installAUT();
-      this.logEvent('appInstalled');
-    }
+    await this.installAUT();
+    this.logEvent('appInstalled');
```

Comment:
we must only log this even is the app was actually present and has been installed